spanner/tests/system.py

```diff
@@ -228,16 +247,49 @@ def test_update_database_ddl(self):
 
         self.assertEqual(len(temp_db.ddl_statements), len(DDL_STATEMENTS))
 
+    def test_db_batch_insert_then_db_snapshot_read_and_db_read(self):
+        from google.cloud.spanner import KeySet
+        keyset = KeySet(all_=True)
```

Comment:
I noticed we make a lot of these in the system tests, could `ALL_KEYS = KeySet(all_=True)` be a global or does it require some context?